CMakeLists.txt

```
@@ -63,7 +63,7 @@ else()
 endif()
 set(Boost_USE_STATIC_RUNTIME OFF)
 
-find_package(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
+find_package(Boost 1.47.0 COMPONENTS chrono OPTIONAL_COMPONENTS system REQUIRED)
```

It looks like OPTIONAL_COMPONENTS was introduced in FindBoost 3.11 (https://github.com/Kitware/CMake/commit/2557cad5ec5084f86d4bb56c82939a49fc9d3071) so may need to try something else given CMake 3.5 minimum.

---

One idea is to run `find_package` twice. I think one of following may work

```suggestion
find_package(Boost 1.47.0 COMPONENTS chrono system QUIET)
if(NOT Boost_FOUND)
  find_package(Boost 1.69.0 COMPONENTS chrono REQUIRED)
endif()
```

```suggestion
find_package(Boost 1.47.0 COMPONENTS chrono REQUIRED)
if(Boost_MAJOR_VERSION EQUAL 1 AND Boost_MINOR_VERSION LESS 69)
  find_package(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
endif()
```